website/blog/2025-01-20-version-0.77.md

```diff
@@ -0,0 +1,395 @@
+---
+title: 'React Native 0.77 - New Styling Features, Android’s 16KB page support, Swift Template'
+authors: [vonovak, mazen, blakef, robhogan, fabriziocucci]
+tags: [engineering]
+date: 2025-01-20
```

Comment:
The date and filename aren't correct anymore. Should it be updated to today?

Reply:
cc @cipolleschi